.env.example

```
@@ -0,0 +1,5 @@
+DB_HOST=localhost
+DB_USER=db_user
+DB_PASSWORD=db_pass
+DB_NAME=db_name
+DB_DIALECT=postgres
```

```suggestion
#server
PORT = 3000

#db
DB_HOST=localhost
DB_USER=db_user
DB_PASSWORD=db_pass
DB_NAME=db_name
DB_DIALECT=postgres
```

Grouped Env variables